config/reports/test-report.jinja2

```diff
@@ -1,18 +1,39 @@
 {# SPDX-License-Identifier: LGPL-2.1-or-later -#}
 
-{{ subject_str }}
+{{ subject }}
 
-Tests Summary
--------------
+Summary
+=======
 
 Tree:     {{ root.revision.tree }}
 Branch:   {{ root.revision.branch }}
 Describe: {{ root.revision.describe }}
 URL:      {{ root.revision.url }}
 SHA1:     {{ root.revision.commit }}
 
-{{'%-15s %s %s'|format("test", "|", "result")}}
-{{'%s%s%s'|format("-"*16, "+", "-"*7)}}
-{%- for test in tests %}
-{{'%-15s %s %s'|format(test.name, "|", test.result)}}
+{{ '%-15s %s %-8s %s %8s %s %8s'|format(
+"Name", "|", "Tesult", "|", "Total", "|", "Failures") }}
```

Comment:
Looks like a typo to me

```suggestion
"Name", "|", "Result", "|", "Total", "|", "Failures") }}
```

Reply:
Tesult is a new word for test result ;)  Or working on a laptop screen...